vnext/Microsoft.ReactNative/Fabric/Composition/CompositionContextHelper.cpp

```diff
@@ -1050,8 +1080,109 @@ struct CompScrollerVisual : winrt::implements<
          0});
   }
 
+  void ConfigureSnapInertiaModifiers() noexcept {
+    auto compositor = m_visual.Compositor();
+
+    // Collect all snap positions
+    std::vector<float> snapPositions;
+
+    if (!m_snapToOffsets.empty()) {
```

Comment:
You dont need this if.  You can just always use this code.  The current logic wouldn't work if m_snapToOffsets is empty, and m_snapToEnd is set.  You wouldn't add the end snap point.

Better to just keep it simpler and have one code path here.